pilot/pkg/networking/core/v1alpha3/sidecar_simulation_test.go

```diff
@@ -2441,8 +2441,10 @@ spec:
 	}
 	for _, variant := range []string{"httproute", "virtualservice"} {
 		t.Run(variant, func(t *testing.T) {
-			for _, tt := range cases {
```

Comment:
this is purely a nit, but you can do `tt := tt` as well to avoid the issue you are working around

Reply:
thank you, done